src/server/scripts/Northrend/zone_wintergrasp.cpp

```diff
@@ -579,6 +580,23 @@ class spell_wintergrasp_defender_teleport_trigger : public SpellScriptLoader
     }
 };
 
+// 58549 Tenacity
+// 59911 Tenacity
```

Comment:
`As of November 16, 2010 and the release of Patch 4.0.3, Tenacity no longer applies to players during the battle.`
Seems this spell was only used during the battle, so we can remove this completely.

Reply:
Spell still exists we gonna leave it for now